pkgs/games/tinyfugue/default.nix

```
@@ -24,11 +29,16 @@ stdenv.mkDerivation rec {
     sha256 = "12fra2fdwqj6ilv9wdkc33rkj343rdcf5jyff4yiwywlrwaa2l1p";
   };
 
+  patches = [
```

If the patch is specific to darwin it may make sense to only apply it when building on darwin:
```suggestion
  patches = lib.optionals stdenv.isDarwin [
```

---

I followed this example on one of my previous PRs: https://github.com/NixOS/nixpkgs/pull/316606#discussion_r1624780278

I think he is right, it's best to have no conditionals so patches failing are easier to catch, and for consistency: both on Linux and Darwin the source code will be the same.

---

Sounds good, thanks for providing the added context, @matteo-pacini, much appreciated.

---

> I think he is right, it's best to have no conditionals so patches failing are easier to catch, and for consistency: both on Linux and Darwin the source code will be the same.

Yes, otherwise updates could easily miss updating the patch for darwin and causing churn.